src/graphql/typeDefs.js

```diff
@@ -19,6 +17,14 @@ module.exports = gql`
     followingCount: Int
   }
 
+  type Feed {
```

Comment:
Rabble calls this a "feed" because it's like a twitter feed i guess. I would personally like to call this something different in the back end because we already have something called a "Feed" (the append only list of messages a particular device publishes)

Reply:
# :hot_pepper: